nncf/pruning/export_helpers.py

```diff
@@ -0,0 +1,394 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+import networkx as nx
+import torch
+
+from nncf.dynamic_graph.graph import NNCFGraph
+from nncf.dynamic_graph.operator_metatypes import NoopMetatype, HardTanhMetatype, TanhMetatype, RELUMetatype, \
+    PRELUMetatype, ELUMetatype, GELUMetatype, SigmoidMetatype, SoftmaxMetatype, AvgPool2dMetatype, MaxPool2dMetatype, \
+    DropoutMetatype, Conv1dMetatype, Conv2dMetatype, Conv3dMetatype, BatchNormMetatype, CatMetatype, AddMetatype, \
+    SubMetatype, DivMetatype, MulMetatype, LinearMetatype, MatMulMetatype, MinMetatype, MaxMetatype, MeanMetatype
+from nncf.nncf_logger import logger as nncf_logger
+from nncf.nncf_network import NNCFNetwork
+from nncf.pruning.export_utils import PruningOperationsMetatypeRegistry, identity_mask_propagation, get_input_masks, \
+    fill_input_masks
+from nncf.pruning.utils import get_sources_of_node
+
+PRUNING_OPERATOR_METATYPES = PruningOperationsMetatypeRegistry("operator_metatypes")
+
+
+# pylint: disable=protected-access
+class DefaultMetaOp:
```

Comment:
It would be great to have a description of the whole idea of the export algorithm [here](https://github.com/openvinotoolkit/nncf_pytorch/blob/master/docs/compression_algorithms/Pruning.md)

Reply:
I see problems with formating when visualizing .md file. Please also consider using header (##) when formatting the text.

Reply:
@AlexKoff88 formatting problems were fixed. Any other comments?